examples/chip-tool/templates/commands.zapt

```diff
@@ -294,6 +300,8 @@ public:
 private:
 {{#if isList}}
     chip::Callback::Callback<{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}ListAttributeCallback> * onSuccessCallback = new chip::Callback::Callback<{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}ListAttributeCallback>(On{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}ListAttributeResponse, this);
+{{else if isStruct}}
+    chip::Callback::Callback<{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}AttributeCallback> * onSuccessCallback = new chip::Callback::Callback<{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}AttributeCallback>(On{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}StructAttributeResponse, this);
```

Comment:
Better would be to fix `ClusterHelper.js` to carry the equivalent of `{{asUpperCamelCase parent.name}}{{asUpperCamelCase name}}` into `{{chipCallback.name}}` and unify all the cases.

Reply:
Agree, would suggest a seperate PR for this since chipCallback.name is widely used (and we need to fix the case for Lists as well of course)

Current the checks for lists in `asChipCallback` looks like:

```
  if (ListHelper.isList(item.type)) {
    return { name : 'List', type : null };
  }
```

Seems not quite correct.

Reply:
A separated PR for that is perfectly fine.